components/hubspot/hubspot.app.mjs

```diff
@@ -667,5 +680,24 @@ export default {
         },
       );
     },
+    async listWorkflows($) {
+      return this.makeRequest(
+        API_PATH.AUTOMATION,
+        "/workflows",
+        {
+          $,
+        },
+      );
+    },
```

Comment:
If the only place this method is called is from `async options`, then there's no need for the `$` param.

```suggestion
    async listWorkflows() {
      return this.makeRequest(
        API_PATH.AUTOMATION,
        "/workflows",
      );
    },
```